Create 2020-05-07-may-meetup.md

LiLietz opened the pull request:
New post for the may 2020 meetup

LiLietz committed:
Create 2020-05-07-may-meetup.md

LiLietz committed:
Update 2020-05-07-may-meetup.md

JoachimK commented on _posts/2020-05-07-may-meetup.md:
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "May 2020 - Remote again! 💪🏼"
+date:   2020-04-29 19:00:00
```
this should be in the past to make sure it's actually included in the generated html (the preview doesn't currently include it)

LiLietz committed:
Update 2020-05-07-may-meetup.md

JoachimK commented on _posts/2020-05-07-may-meetup.md:
```diff
@@ -0,0 +1,57 @@
+
+[Meetup RSVP](https://www.meetup.com/CocoaHeads-Hamburg/events/zkldgrybchbkb/)
+
+#### Note on Zoom
+Zoom has a [history of privacy violations and generally shady behavior](https://twitter.com/dhh/status/1244997990382596096). However, it is also currently the most easy-to-setup tool for a group of our size. We'll look for alternatives for future meetups, but until then, we advise you *not to install Zoom on your Mac*. We believe the iOS client to be sufficiently safe due to sandboxing. You can a ...
```
let's remove this for the time and add it back in if we do go with Zoom.

LiLietz committed:
Rename 2020-05-07-may-meetup.md to 2020-04-28-may-meetup.md

LiLietz committed:
Update 2020-04-28-may-meetup.md

LiLietz committed:
Hide post publishing date from the home page

We don't need to show the date of when the post was published, right?

LiLietz committed:
Update 2014-08-01-january-meetup.md

LiLietz committed:
Update 2013-12-04-welcome-to-jekyll.markdown

JoachimK commented on _posts/2020-04-28-may-meetup.md:
```diff
@@ -0,0 +1,54 @@
+
+As our last months online event worked very well 🎉 we are going to meet remotely again. We are currently evaluating different video call tools to see which one to use and will update here in time.
+
+Also, advices and tipps from you are very welcome! 🙏🏼
```
I think "advice" doesn't have a plural, so it should loose the "s" and it's "tips" in English (but "Tipps" in German... yeah... :D )

JoachimK commented on _posts/2020-04-28-may-meetup.md:
```diff
@@ -0,0 +1,54 @@
+
+Let's gather in Slack!
+
+Click here 👉🏼 https://slack.cocoaheads.hamburg
+to join our workspace 'cocoaheadshamburg.slack.com' to enable an easier communication.
```
Let's make these proper links here, we only don't do it in Meetup as Meetup doesn't support it.

JoachimK commented on _posts/2020-04-28-may-meetup.md:
```diff
@@ -0,0 +1,54 @@
+Everyone joins the video call.
+
+19:45 - 1️⃣ Talk: Learning Combine by refactoring a networking layer - Donny Walls
+In this talk Donny will explain the basics of Combine, and how they can be applied to refactor an existing networking layer into one that uses Combine. Attendees will learn about Combine’s Publishers, Subscribers, operators, the Cancellable protocol and more.
```
You'll need an empty line before this if you want the line-break in the result.

Same for the 2nd talk description

LiLietz committed:
Only show last post content

Only show last post content
In Older Posts show older posts from current year
In Archive show posts from previous year and older

LiLietz committed:
Hide publishing date from old posts and archived

LiLietz committed:
Update 2020-04-28-may-meetup.md

LiLietz committed:
Merge branch 'master' into 2020-may-meetup

LiLietz committed:
Hide publishing date from post view

LiLietz committed:
try base font size for post links

LiLietz committed:
Reduce font size for old posts and archive lists

LiLietz committed:
revert font size changes

lilietz-lyft committed:
add cocoaheads banner

lilietz-lyft committed:
rename banner image

LiLietz committed:
Update index.md

lilietz-lyft committed:
crop banner

LiLietz committed:
Small introduction text

LiLietz committed:
separate introduction from the posts

LiLietz committed:
make home page sections to distanciate

LiLietz committed:
test margin for home-section class

LiLietz committed:
test headlines color

LiLietz committed:
test some styling

LiLietz committed:
style post-link

LiLietz committed:
style post-link

LiLietz committed:
Update home.html

LiLietz committed:
Update home.html

LiLietz committed:
Update _layout.scss

LiLietz committed:
Update home.html
